lib/stream.js

```diff
@@ -39,14 +45,21 @@ Stream.PassThrough = require('_stream_passthrough');
 Stream.pipeline = pipeline;
 Stream.finished = eos;
 
-Stream.promises = promises;
+ObjectDefineProperty(Stream, 'promises', {
+  configurable: true,
+  enumerable: true,
+  get() {
+    if (!promises) promises = require('internal/streams/promises');
+    return promises;
+  }
+});
 
 // Backwards-compat with node 0.4.x
 Stream.Stream = Stream;
 
 Stream._isUint8Array = require('internal/util/types').isUint8Array;
 Stream._uint8ArrayToBuffer = function _uint8ArrayToBuffer(chunk) {
   return new internalBuffer.FastBuffer(chunk.buffer,
-                                       chunk.byteOffset,
-                                       chunk.byteLength);
+    chunk.byteOffset,
+    chunk.byteLength);
```

Comment:
unrelated?

Reply:
oh... I accidentally formatted the file.